esmvaltool/cmorizers/obs/cmor_config/ERA-Interim.yml

```diff
@@ -0,0 +1,238 @@
+    short_name: rss
+    mip: day
+    raw: ssr
+    file: 'ERA-Interim_ssr_daily_*.nc'
+  tsns:
+    mip: Eday
+    raw: tsn
+    file: 'ERA-Interim_tsn_daily_*.nc'
+  rsdt_day:
+    short_name: rsdt
+    mip: CFday
+    raw: tisr
+    file: 'ERA-Interim_tisr_daily_*.nc'
+  clt_day:
+    short_name: clt
+    mip: day
+    raw: tcc
+    file: 'ERA-Interim_tcc_daily_*.nc'
+  zg_day:
+    short_name: zg
+    mip: day
+    raw: z
+    file: 'ERA-Interim_z_daily_*.nc'
+  pr_month:
+    short_name: pr
+    mip: Amon
+    raw: tp
+    file: 'ERA-Interim_tp_monthly_*.nc'
+  clivi:
+    mip: Amon
+    raw: p57.162
+    file: 'ERA-Interim_p57.162_monthly_*.nc'
+  clt_month:
+    short_name: clt
+    mip: Amon
+    raw: tcc
+    file: 'ERA-Interim_tcc_monthly_*.nc'
+  prw:
+    mip: Amon
+    raw: tcwv
+    file: 'ERA-Interim_tcwv_monthly_*.nc'
+  psl_month:
+    short_name: psl
+    mip: Amon
+    raw: msl
+    file: 'ERA-Interim_msl_monthly_*.nc'
+  ps:
+    mip: Amon
+    raw: sp
+    file: 'ERA-Interim_sp_monthly_*.nc'
+  tas_month:
+    short_name: tas
+    mip: Amon
+    raw: t2m
+    file: 'ERA-Interim_t2m_monthly_*.nc'
+  tauu:
+    mip: Amon
+    raw: iews
+    file: 'ERA-Interim_iews_monthly_*.nc'
+  tauv:
+    mip: Amon
+    raw: inss
+    file: 'ERA-Interim_inss_monthly_*.nc'
+  tos:
+    mip: Omon
+    raw: sst
+    file: 'ERA-Interim_sst_monthly_*.nc'
+  ts_month:
+    short_name: ts
+    mip: Amon
+    raw: skt
+    file: 'ERA-Interim_skt_monthly_*.nc'
+  zg_month:
+    short_name: zg
+    mip: Amon
+    raw: z
+    file: 'ERA-Interim_z_monthly_*.nc'
+  ta:
+    mip: Amon
+    raw: t
+    file: 'ERA-Interim_t_monthly_*.nc'
+  ua:
+    mip: Amon
+    raw: u
+    file: 'ERA-Interim_u_monthly_*.nc'
+  va:
+    mip: Amon
+    raw: v
+    file: 'ERA-Interim_v_monthly_*.nc'
+  wap:
+    mip: Amon
+    raw: w
+    file: 'ERA-Interim_w_monthly_*.nc'
+  hus:
+    mip: Amon
+    raw: q
+    file: 'ERA-Interim_q_monthly_*.nc'
+  hur:
+    mip: Amon
+    raw: r
+    file: 'ERA-Interim_r_monthly_*.nc'
+  evspsbl_month:
+    short_name: evspsbl
+    mip: Amon
+    raw: e
+    file: 'ERA-Interim_e_monthly_*.nc'
+  uas_month:
+    short_name: uas
+    mip: Amon
+    raw: u10
+    file: 'ERA-Interim_u10_monthly_*.nc'
+  vas_month:
+    short_name: vas
+    mip: Amon
+    raw: v10
+    file: 'ERA-Interim_v10_monthly_*.nc'
+  tdps_month:
+    short_name: tdps
+    mip: Emon
+    raw: d2m
+    file: 'ERA-Interim_d2m_monthly_*.nc'
+  prsn_month:
+    short_name: prsn
+    mip: Amon
+    raw: sf
+    file: 'ERA-Interim_sf_monthly_*.nc'
+  rss_month:
+    short_name: rss
+    mip: Emon
+    raw: ssr
+    file: 'ERA-Interim_ssr_monthly_*.nc'
+  rsds_month:
+    short_name: rsds
+    mip: Amon
+    raw: ssrd
+    file: 'ERA-Interim_ssrd_monthly_*.nc'
+  rsdt_month:
+    short_name: rsdt
+    mip: Amon
+    raw: tisr
+    file: 'ERA-Interim_tisr_monthly_*.nc'
+  tsn:
+    mip: LImon
+    raw: tsn
+    file: 'ERA-Interim_tsn_monthly_*.nc'
+# TODO handle multi raw
```

Comment:
If we get rid of the NCL cmorizer, we should make sure that all variables are ported, including these.

Reply:
@mattiarighi Thanks. Regarding these variables, the variables setting in NCL file is mentioned as: 
[/"clwvi",  (/"p56.162", "p57.162"/),         1.,     "Amon", "mon"/], \
[/"hfds",   (/"ssr", "str", "slhf", "sshf"/), 1.,     "Omon", "mon"/], \

I couldn't find any function/source to obtain, for example "clwvi" from other two parameters i.e. (/"p56.162", "p57.162"/). In other words, it is not clear how the NCL script handle the two variables "clwvi" and "hfds".  Please let me know if I missed something.

Reply:
This is done at lines 224-242 in the ncl script.

Basically, you need to sum over the variables given as array in the element 1 of the list. 
So for clwvi, this would be:
```
clwvi = p56.162 + p57.162
```

Hope this helps.

Reply:
@mattiarighi  Thank you. The multiple case is added to support all the variables in NCL script.